frontend/device/kobo/device.lua

```diff
@@ -739,9 +739,33 @@ local function writeToSys(val, file)
     return re
 end
 
---- put device into standby, touchscreen is enabled
+--- Check if device can really go to standby.
+-- Here we do a dance. At start Kobo.standby is set to Kobo._testStandby
+-- On the first call `/sys/power/state` is checked if `standby` is available:
+--    If so set Kobo.standby to Kobo._realStandby, which is used on further calls.
+--    If `standby` is not available disable `self.canStandby`.
+function Kobo:_testStandby()
```

Comment:
Err, everything should support `mem` (which is what `standby` is an alias of).

Reply:
I found that shallow is a synonym of standby, whereas mem is synonym to deep.
But I did not find anything on dr google that mem=standby